### PR TITLE
Update using-fonts.md

### PR DESCRIPTION
**Pull request by abrarpathan19**
## Purpose of this pull request

This pull request (PR) will correct the fonts format eot not oet.

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.4/frontend-dev-guide/css-topics/using-fonts.html
## Links to Magento source code

-  ...

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/frontend-dev-guide/css-topics/using-fonts.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ To ensure the stability of your customizations and prevent upgrades from overwri
    }
    ```
 
-   `@font-format` is optional. If you need to specify a format, enclose it in single quotes. For example, `@font-format: 'ttf'` for TrueType Fonts. The available types are: "woff", "woff2", "ttf", "oet", "otf", and "svg"
```
Comment (BarnyShergold) [inline src/guides/v2.3/frontend-dev-guide/css-topics/using-fonts.md:50]: Should this not also be corrected in - src/guides/v2.3/frontend-dev-guide/css-topics/using-fonts.md

**Review by BarnyShergold — CHANGES_REQUESTED** [review]

**Review by BarnyShergold — APPROVED** [review]

**Comment by BarnyShergold** [comment]
running tests

**Comment by BarnyShergold** [comment]
@magento run all tests

**Review by jeff-matthews — APPROVED** [review]

**Comment by dobooth** [comment]
running tests

**Pull request merged**